### ISSUE TITLE
ECR Python SDK did not generate valid syntax

### ISSUE THREAD

**Issue by inhumantsar** [pulumi/pulumi#9425]
## Hello!

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

## Issue details

```
...
  File "/Users/jingbofu/.pyenv/versions/mario/lib/python3.9/site-packages/pulumi_aws_native/ecr/__init__.py", line 10, in <module>
    from .get_pull_through_cache_rule import *
  File "/Users/jingbofu/.pyenv/versions/mario/lib/python3.9/site-packages/pulumi_aws_native/ecr/get_pull_through_cache_rule.py", line 23
    class AwaitableGetPullThroughCacheRuleResult(GetPullThroughCacheRuleResult):
IndentationError: expected an indented block
```

There are significant syntax errors in the named module:

* https://github.com/pulumi/pulumi-aws-native/blob/master/sdk/python/pulumi_aws_native/ecr/get_pull_through_cache_rule.py#L19-L21
* https://github.com/pulumi/pulumi-aws-native/blob/master/sdk/python/pulumi_aws_native/ecr/get_pull_through_cache_rule.py#L28-L29
* https://github.com/pulumi/pulumi-aws-native/blob/master/sdk/python/pulumi_aws_native/ecr/get_pull_through_cache_rule.py#L48-L51

### Steps to reproduce

1. Install 0.13.0
2. Create ECR repo with this module

Expected: no 💥
Actual: 💥

In this situation, one of our guys did a `pip install -U` and got the latest version. Our projects are currently pinned to 0.9.0 as it is known-working, but I'm really surprised this passed testing as it seems like this module shouldn't even import without throwing an exception.

**Comment by Frassle** [comment]
This looks like a python codegen bug, moving to pulumi/pulumi.

**Comment by inhumantsar** [comment]
Two months and several point releases later this is still an issue. 

I get that the native module is in preview, but this is really hard on the credibility. Every day this goes without getting fixed is an extra week advocates need to spend making excuses for it

**Comment by justinvp** [comment]
The Python SDK codegen issue was fixed in #9649, and fixed in 0.17.0 of the aws-native provider and the corresponding release of cf2pulumi.
